frontend/src/app/modules/exclusion-dates/exclusion-dates-picker/exclusion-dates-picker.component.ts

```diff
@@ -70,6 +71,11 @@ export class ExclusionDatesPickerComponent extends BaseComponent implements OnIn
             }))
             .filter((range) => range.start && range.end);
 
+    updateStartRangeValidators() {
+        this.timeControlsIdentifiers
+            .forEach((number) => this.formGroup.get(number.toString() + this.startRangeIdentifier)?.updateValueAndValidity());
+    }
```

Comment:
Can not we use a map?

Reply:
We just need to call updateValueAndValidity for each control and there is no need for map